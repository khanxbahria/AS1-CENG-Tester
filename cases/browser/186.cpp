#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Browser* brows = new Browser();
Window a;
Window b;
Window c;
Window d;
Tab A = Tab("w", "e","hh");
Tab B = Tab("uu", "o","dd");
Tab C = Tab("mm", "ppp","xxx");
Tab D = Tab("qq", "uu","t");
Tab E = Tab("o", "mm","yyy");
Tab F = Tab("ggg", "bb","ww");
Tab G = Tab("ddd", "zz","y");
Tab H = Tab("p", "ii","qq");
Tab I = Tab("y", "ccc","pp");
Tab J = Tab("rr", "ss","m");
Tab K = Tab("jjj", "fff","hh");
Tab L = Tab("rrr", "ggg","a");
Tab M = Tab("iii", "tt","aaa");
Tab N = Tab("ttt", "fff","e");
Tab O = Tab("u", "cc","cc");
Tab P = Tab("kkk", "h","g");
Tab Q = Tab("p", "z","ww");
Tab R = Tab("qq", "www","mmm");
Tab S = Tab("rr", "fff","v");
Tab T = Tab("z", "hh","c");
Tab U = Tab("r", "pp","ll");
Tab V = Tab("rrr", "nnn","vvv");
Tab W = Tab("i", "m","mmm");
Tab X = Tab("l", "c","bb");
Tab Y = Tab("nn", "ee","nnn");
Tab Z = Tab("sss", "sss","nn");
a.newTab(A);
b.changeActiveTabTo(0);
c.newTab(B);
d.changeActiveTabTo(0);
a.newTab(C);
b.changeActiveTabTo(4);
c.newTab(D);
d.moveActiveTabTo(4);
a.newTab(E);
b.changeActiveTabTo(2);
c.newTab(F);
d.changeActiveTabTo(2);
a.changeActiveTabTo(5);
b.moveActiveTabTo(5);
c.changeActiveTabTo(5);
d.newTab(G);
a.moveActiveTabTo(5);
b.newTab(H);
c.newTab(I);
d.changeActiveTabTo(5);
a.newTab(J);
b.newTab(K);
c.changeActiveTabTo(0);
d.newTab(L);
a.changeActiveTabTo(1);
b.changeActiveTabTo(1);
c.moveActiveTabTo(1);
d.newTab(M);
a.newTab(N);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.changeActiveTabTo(3);
a.changeActiveTabTo(0);
b.changeActiveTabTo(0);
c.changeActiveTabTo(0);
d.newTab(O);
a.newTab(P);
b.newTab(Q);
c.moveActiveTabTo(5);
d.moveActiveTabTo(5);
a.moveActiveTabTo(3);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.moveActiveTabTo(3);
a.moveActiveTabTo(2);
b.changeActiveTabTo(2);
c.moveActiveTabTo(2);
d.moveActiveTabTo(2);
a.changeActiveTabTo(5);
b.changeActiveTabTo(5);
c.newTab(R);
d.newTab(S);
a.moveActiveTabTo(4);
b.moveActiveTabTo(4);
c.newTab(T);
d.changeActiveTabTo(4);
a.moveActiveTabTo(2);
b.newTab(U);
c.moveActiveTabTo(2);
d.newTab(V);
a.moveActiveTabTo(1);
b.changeActiveTabTo(1);
c.changeActiveTabTo(1);
d.newTab(W);
a.changeActiveTabTo(0);
b.changeActiveTabTo(0);
c.changeActiveTabTo(0);
d.changeActiveTabTo(0);
a.moveActiveTabTo(1);
b.newTab(X);
c.changeActiveTabTo(1);
d.moveActiveTabTo(1);
a.moveActiveTabTo(1);
b.newTab(Y);
c.moveActiveTabTo(1);
d.changeActiveTabTo(1);
a.changeActiveTabTo(1);
b.newTab(Z);
c.newTab(Z);
d.moveActiveTabTo(1);
b.changeActiveTabTo(1);
d.moveActiveTabTo(1);
a.changeActiveTabTo(0);
b.moveActiveTabTo(0);
c.moveActiveTabTo(0);
a.changeActiveTabTo(0);
c.moveActiveTabTo(0);
d.changeActiveTabTo(0);
b.moveActiveTabTo(3);
d.changeActiveTabTo(3);
b.changeActiveTabTo(4);
c.changeActiveTabTo(4);
d.moveActiveTabTo(4);
a.changeActiveTabTo(4);
d.changeActiveTabTo(4);
a.changeActiveTabTo(0);
c.changeActiveTabTo(0);
d.moveActiveTabTo(0);
b.moveActiveTabTo(4);
b.moveActiveTabTo(2);
c.changeActiveTabTo(2);
d.moveActiveTabTo(2);
a.changeActiveTabTo(0);
c.changeActiveTabTo(0);
a.changeActiveTabTo(5);
d.moveActiveTabTo(5);
a.changeActiveTabTo(4);
b.changeActiveTabTo(4);
d.changeActiveTabTo(4);
b.changeActiveTabTo(3);
c.moveActiveTabTo(3);
a.moveActiveTabTo(4);
b.moveActiveTabTo(4);
d.changeActiveTabTo(4);
a.moveActiveTabTo(0);
b.moveActiveTabTo(0);
a.changeActiveTabTo(5);
c.changeActiveTabTo(5);
d.changeActiveTabTo(5);
c.moveActiveTabTo(5);
d.moveActiveTabTo(5);
a.moveActiveTabTo(0);
a.changeActiveTabTo(4);
b.moveActiveTabTo(4);
d.changeActiveTabTo(4);
a.changeActiveTabTo(1);
b.moveActiveTabTo(1);
d.moveActiveTabTo(1);
a.changeActiveTabTo(1);
b.moveActiveTabTo(1);
c.moveActiveTabTo(1);
d.moveActiveTabTo(1);
a.changeActiveTabTo(3);
d.moveActiveTabTo(3);
a.changeActiveTabTo(1);
b.moveActiveTabTo(1);
d.changeActiveTabTo(1);
b.changeActiveTabTo(5);
c.moveActiveTabTo(5);
d.moveActiveTabTo(5);
c.changeActiveTabTo(0);
d.moveActiveTabTo(0);
c.moveActiveTabTo(3);
d.changeActiveTabTo(3);
a.moveActiveTabTo(2);
b.changeActiveTabTo(2);
c.changeActiveTabTo(2);
d.changeActiveTabTo(2);
a.changeActiveTabTo(0);
b.changeActiveTabTo(0);
c.moveActiveTabTo(0);
d.moveActiveTabTo(0);
b.changeActiveTabTo(1);
c.changeActiveTabTo(1);
a.changeActiveTabTo(1);
d.moveActiveTabTo(1);
brows->addWindow(a);
brows->addWindow(b);
brows->addWindow(c);
brows->addWindow(d);
brows->print();
brows->closeEmptyWindows();
brows->newWindow();
brows->switchToWindow(9);
brows->getWindow(8);
brows->closeWindow();
brows->switchToWindow(8);
brows->mergeAllWindows();
brows->closeWindow();
brows->print();
brows->print();
brows->closeWindow();
brows->getWindow(1);
brows->newWindow();
brows->print();
brows->newWindow();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->closeWindow();
brows->switchToWindow(7);
brows->getWindow(8);
brows->mergeAllWindows();
brows->switchToWindow(6);
brows->closeWindow();
brows->print();
brows->closeEmptyWindows();
brows->closeWindow();
brows->newWindow();
brows->newWindow();
brows->print();
brows->closeEmptyWindows();
brows->newWindow();
brows->closeEmptyWindows();
brows->print();
brows->closeWindow();
brows->print();
brows->switchToWindow(5);
brows->closeWindow();
brows->print();
brows->mergeAllWindows();
brows->print();
brows->switchToWindow(4);
brows->newWindow();
brows->mergeAllWindows();
brows->print();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->getWindow(1);
brows->print();
brows->print();
brows->switchToWindow(0);
brows->switchToWindow(3);
brows->switchToWindow(0);
brows->newWindow();
brows->getWindow(5);
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->switchToWindow(4);
brows->getWindow(9);
brows->print();
brows->getWindow(3);
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->switchToWindow(2);
brows->getWindow(7);
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->switchToWindow(8);
brows->newWindow();
brows->getWindow(9);
brows->switchToWindow(0);
brows->closeWindow();
brows->getWindow(3);
brows->newWindow();
brows->print();
brows->newWindow();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->getWindow(6);
brows->switchToWindow(9);
brows->switchToWindow(8);
brows->closeWindow();
brows->closeEmptyWindows();
brows->print();
brows->print();
brows->newWindow();
brows->mergeAllWindows();
brows->switchToWindow(5);
brows->print();
brows->newWindow();
brows->switchToWindow(8);
brows->switchToWindow(7);
brows->newWindow();
brows->newWindow();
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->closeWindow();
brows->closeEmptyWindows();
brows->newWindow();
brows->switchToWindow(0);
brows->getWindow(3);
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->switchToWindow(2);
brows->getWindow(5);
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->getWindow(7);
brows->closeEmptyWindows();
brows->print();
brows->getWindow(8);
brows->getWindow(5);
brows->getWindow(6);
brows->switchToWindow(5);
brows->getWindow(0);
brows->print();
brows->print();
brows->closeEmptyWindows();
brows->print();
delete brows;
}
	return 0;
}