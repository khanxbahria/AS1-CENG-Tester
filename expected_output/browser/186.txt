The active tab is tab 0




The active tab is tab 1
ppp
mm
xxx

mm
o
yyy

fff
ttt
e





e
w
hh

h
kkk
g

ss
rr
m

The active tab is tab 1




sss
sss
nn

ee
nn
nnn

fff
jjj
hh

c
l
bb

z
p
ww

ii
p
qq

pp
r
ll

The active tab is tab 1
sss
sss
nn

uu
qq
t

hh
z
c





ccc
y
pp

www
qq
mmm

bb
ggg
ww

o
uu
dd

The active tab is tab 1




cc
u
cc

zz
ddd
y

nnn
rrr
vvv

ggg
rrr
a

fff
rr
v

m
i
mmm

tt
iii
aaa

Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 0




Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
The browser is empty
The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 1








Window is empty
The list is empty.
The active tab is tab 1








The active tab is tab 1








The active tab is tab 2












The active tab is tab 0




The active tab is tab 2












The active tab is tab 0




The active tab is tab 2












The active tab is tab 0




The active tab is tab 2












The active tab is tab 4




















Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




